#! /bin/sh /usr/share/dpatch/dpatch-run
## 40_help_browser.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
##
## DP: Use sensible-browser instead of dillo for displaying the help text.

@DPATCH@
Index: roadmap/src/preferences
===================================================================
RCS file: /cvsroot/roadmap/roadmap/src/preferences,v
retrieving revision 1.6
diff -u -3 -p -u -r1.6 preferences
--- roadmap/src/preferences	29 Nov 2004 07:02:09 -0000	1.6
+++ roadmap/src/preferences	26 Feb 2005 12:15:26 -0000
@@ -12,6 +12,7 @@ GPS.Color: red
 GPS.Foreground: black
 GPS.Source: gpsd://localhost
 GPS.Timeout: 10
+Help.Browser: sensible-browser
 Highlight.Background: yellow
 Highlight.Duration: 10
 Highlight.Thickness: 4
